### DISCUSSION TITLE
Selectable Entries Condition: "Empty" and "Not Empty"

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#11483] [Ideas] [answered]
It would be great to have the dropdown condition "empty" and "not empty" for example when setting this against the URI so only entries that have a URI can be selected.

<img width="364" alt="CleanShot 2022-06-22 at 23 38 46@2x" src="https://user-images.githubusercontent.com/2505643/175043798-2fd71696-c498-4f50-b1c7-49c3654e56c7.png">

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested by #11070. You can follow along there :)
